### PR TITLE
jQuery 3 follow ups: fix merge page and subjects error

### PR DESCRIPTION
**Pull request by jdlrobson**
Since upgrading to jQuery 3 feature, the browser detection is no
longer available. Given we support IE >=11 which supports opacity (filter was used < IE11 to do that)

this code is redundant and can be removed.

I get a JS error `Uncaught TypeError: t.browser is undefined` on http://localhost:8080/subjects/mathematics
The error resolves with this patch.

### Technical

Missed in the jQuery 3 upgrade.

### Testing
* Make sure no JavaScript errors on  http://localhost:8080/subjects/mathematics
* Create a new author Mark Twain 2 while logged in as an admin. Go to http://localhost:8080/search/authors?q=Mark+Twain and click merge authors. Click "Merge authors".  Make sure no javaScript error displays.
* 
### Stakeholders
@mekarpeles @cdrini  - deploy blocker I suspect.

### CODE REVIEW

**Comment by mekarpeles** [comment]
lgtm, adding to testing.

File: openlibrary/plugins/openlibrary/js/jquery.customFade.js [hunk 1]
```diff
@@ -4,8 +4,6 @@ export default function addFadeInFunctionsTojQuery($) {
     // books/edit/edition.html, publishers/view.html and subjects.html
```
Comment (cdrini) [inline openlibrary/plugins/openlibrary/js/jquery.customFade.js:4]: Future PR: We can now delete this entire file and use the stock jquery fadein/fadeout!

**Review by cdrini — APPROVED** [review]
Confirmed in local dev env that merging works as expected.

File: openlibrary/plugins/openlibrary/js/jquery.customFade.js (hunk 1 above)
Comment (jdlrobson) [reply openlibrary/plugins/openlibrary/js/jquery.customFade.js:4]: ohhh yeh. nice!

Comment (cdrini) [reply openlibrary/plugins/openlibrary/js/jquery.customFade.js:4]: https://github.com/internetarchive/openlibrary/issues/5213

**Pull request merged**